common/templates/layouts/collection.njk

```diff
@@ -6,36 +6,50 @@
   }) }}'});
 {% endblock %}
 
-{% if period === 'week' %}
-  {% set datetime = dateRange | formatISOWeek %}
-  {% set displayDate = dateRange | formatDateRangeAsRelativeWeek %}
-{% else %}
-  {% set datetime = dateRange | first %}
-  {% set displayDate = datetime | formatDateAsRelativeDay %}
-{% endif %}
-
 {% block pageTitle %}
-  {{ t("collections::page_title_with_date", {
-    context: context,
-    date: displayDate | lower
-  }) }}
+  {% if period === 'week' %}
+    {{ t("collections::page_title_with_date", {
+      context: context,
+      date: dateRange | formatDateRangeAsRelativeWeek
+    }) }}
+  {% else %}
+    {{ t("collections::page_title_with_date", {
+      context: context,
+      date: datetime | formatDateAsRelativeDay
+    }) }}
+  {% endif %}
 {% endblock %}
 
 {% block content %}
 
   {% block pageHeader %}
     <header class="govuk-grid-row govuk-!-margin-bottom-7">
       <div class="govuk-grid-column-two-thirds">
-        <span class="govuk-caption-xl">
+        <h1 class="govuk-heading-xl govuk-!-margin-bottom-2">
+          <span class="govuk-caption-xl">
           {{ t("collections::subheading", {
             context: context
           }) }}
-        </span>
-        <h1 class="govuk-heading-xl govuk-!-margin-bottom-2">
-          {{ appTime({
-            datetime: datetime,
-            text: displayDate
-          }) }}
+          </span>
+
+          <span class="govuk-!-display-block">
+          {% if period === 'week' %}
+            {{ dateRange | formatDateRange }}
+            {{ appTime({
+              datetime: dateRange | formatISOWeek,
+              text: '(' + dateRange | formatDateRangeAsRelativeWeek | lower + ')',
+              classes: 'app-secondary-text-colour'
+              }) }}
+          {% else %}
+            {% set datetime = dateRange | first %}
+            {{ datetime | formatDateWithDay }}
```

Comment:
So it looks like we've lost the day from the original format and it now also displays the date in grey on every page:

this
![image](https://user-images.githubusercontent.com/3327997/89623511-055e1500-d895-11ea-8e62-e1631db8e0de.png)

should be this:
![image](https://user-images.githubusercontent.com/3327997/89623677-5837cc80-d895-11ea-99d6-1dff28f04d57.png)

and this:
![image](https://user-images.githubusercontent.com/3327997/89623722-6980d900-d895-11ea-8335-b69b6e1fe8ee.png)

should be this:
![image](https://user-images.githubusercontent.com/3327997/89623754-77365e80-d895-11ea-8769-ee56acd41e0a.png)